docs/gpu-acceleration.md

```diff
@@ -239,15 +255,16 @@ isn't the case, these steps might help to enable it.
 `torch` should already ship with Metal support, so you only have to
 replace `llama-cpp-python`. Go to the project's GitHub to see the
 [supported backends](https://github.com/abetlen/llama-cpp-python?tab=readme-ov-file#supported-backends).
-Find the `Metal` backend. You'll see a `pip3 install` command. You'll want to
+Find the `Metal` backend. You'll see a `pip install` command. You'll want to
 add a few options to ensure it gets installed over the existing package:
 `--force-reinstall` and `--no-cache-dir`. Your final command should look like so:
```

Comment:
There are still several mentions of `--no-cache-dir` remaining in the doc. Did you mean to remove them all?

Reply:
Yes, in the near future. The GPU acceleration doc needs a general overhaul / rewrite anyway.